content/showcase/atelierehoi/index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Atelier EHOI"
+description: "Atelier EHOI is a New York and New Jersey-based online dessert shop that delivers freshly baked sweets to your doorsteps."
+lead: "Atelier EHOI is a New York and New Jersey-based online dessert shop that delivers freshly baked sweets to your doorsteps."
```

Comment:
why is the lead and description different ?

Reply:
they're the same